### DISCUSSION TITLE
date and time modification filtering

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#2173] [Q&A] [answered]
![pesquisa_por_data_horario](https://github.com/sepinf-inc/IPED/assets/66680566/7dd1cf2e-15b2-47a5-8ad3-34a9128c569a)

As shown in the attached image, I'm trying to filter the file modification date by date and time, but in the time part the iped is not matching. Am I doing something wrong?

**Comment by lfcnassif** [comment, marked as answer]
No, your syntax is correct. Just tested it here, but using the field english name (modified) and it works. Not sure what is going on within your case...

**Comment by wladimirleite** [comment]
I believe there is a bug related to localized properties. 
It works from "actual" field names (like "modified"), but not with the localized version.
@gisms, try using "modified" instead (at least for now). 

PS: I am trying to fix it.

**Reply by gisms** [reply]
It worked with modified!!!

Thanks a lot
